src/components/compute-credits/ComputeCreditsBuyDialog.js

```diff
@@ -175,7 +175,7 @@ class ComputeCreditsBuyDialog extends React.Component {
         </DialogContent>
         <DialogActions>
           <Button onClick={this.handleBuyCredits} disabled={this.state.makingPayment} color="primary" variant="contained">
-            Buy {credits.toLocaleString("en-US", {useGrouping: true})} credits
+            Buy {credits.toLocaleString("en-US", {useGrouping: true})} credits with browser
```

Comment:
IMO "with browser" doesn't sound right to me. [Google search](https://www.google.com/search?q=%22buy+*+with+browser%22&oq=%22buy+*+with+browser%22&aqs=chrome..69i57j0.11800j0j7&sourceid=chrome&ie=UTF-8) also doesn't show much results.

Reply:
What do you think sounds better?

Reply:
Like it is right now without clarification. 🤔